test/integration/admin/api/objects_controller_test.rb

```diff
@@ -0,0 +1,80 @@
+require 'test_helper'
+
+class Admin::Api::ObjectsControllerTest < ActionDispatch::IntegrationTest
+
+  CONTROLLER = Admin::Api::ObjectsController
+
+  def setup
+    @provider = FactoryBot.create(:provider_account)
+
+    host! @provider.admin_domain
+
+    login_provider @provider
+  end
+
+  def test_status_404
+    get admin_api_objects_status_path(object_type: 'service', object_id: 1)
```

Comment:
Is this a service that does not exist or that belongs to another tenant? I think that we should test both. And I also think that it is risky to test it with a hardcoded ID number because it may vary in the tests (specially in Oracle)

Reply:
hardcoded ID, true true.